docs/main/faqs/faqs.md

```diff
@@ -591,7 +591,7 @@ The Replicated Storage nightly builds and releases are compiled and tested on x8
 
 Minimum hardware requirements are discussed in the [quickstart section](../quickstart-guide/installation.md) of this documentation.
 
-Replicated Storage does not run on Raspbery Pi as the version of the SPDK. Replicated Storage requires ARMv8 Crypto extensions which are not currently available for Pi.
+Replicated Storage does not run on Raspbery Pi as the current version of SPDK requires ARMv8 Crypto extensions. Replicated Storage requires ARMv8 Crypto extensions which are not currently available for Pi.
```

Comment:
which are not currently available for Pi?

Reply:
This is alright, as the "are" refers to "ARMv8 Crypto extensions" which is in the plural form.

Reply:
I meant to combine the two sentences as follows:
Replicated Storage does not run on Raspbery Pi as the current version of SPDK requires ARMv8 Crypto extensions which are not currently available for Pi.

Reply:
got it. done.